package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s47_most_searched_website_on-google",
+  "version": "1.0.0",
+  "description": "Idea title : Most googled website in 2023",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
```

Comment:
Consider adding relevant keywords to the "keywords" field to improve the project's discoverability. [medium]